pkg/diagnostics/README.md

```diff
@@ -0,0 +1,113 @@
+OpenShift v3 Diagnostics
+========================
+
+This is a tool to help administrators and users resolve common problems
+that occur with OpenShift v3 deployments. It will likely remain
+under continuous development as the OpenShift Origin project progresses.
+
+The goals of the diagnostics tool are summarized in this [Trello
+card](https://trello.com/c/LdUogKuN). Diagnostics are included as an
+`openshift` binary sub-command that analyzes OpenShift as it finds it,
+whether from the perspective of an OpenShift client or on an OpenShift
+host.
+
+Expected environment
+====================
+
+OpenShift can be deployed in many ways: built from source, included
+in a VM image, in a Docker image, or as enterprise RPMs. Each of these
+would imply different configuration and environment. In order to keep
+assumptions about environment to a minimum, the diagnostics have been
+added to the `openshift` binary itself so that wherever there is an
+OpenShift server or client, the diagnostics can run in the exact same
+environment.
+
+Diagnostics looks for config files in standard locations. If not found,
+related diagnostics are just skipped. Non-standard locations can be
+specified with flags.
+
+Host environment
+================
+
+Master/node diagnostics will be most useful in a specific target
+environment, which is a deployment using RPMs and ansible deployment
+logic. This provides two major benefits:
+
+* master/node configuration is based on a configuration file in a standard location
+* all components log to journald
+
+Having configuration files in standard locations means you will generally
+not even need to specify where to find them. Running:
+
+    openshift ex diagnostics
```

Comment:
it's a little odd for diagnostics to default to paths that `openshift start` doesn't even default to